src/coreclr/src/tools/crossgen2/ILCompiler.ReadyToRun/Compiler/DependencyAnalysis/ReadyToRunCodegenNodeFactory.cs

```diff
@@ -408,7 +405,7 @@ public IMethodNode MethodEntrypoint(MethodWithToken method, bool isUnboxingStub,
 
         private NodeCache<MethodDesc, MethodWithGCInfo> _localMethodCache;
 
-        private MethodWithGCInfo CreateMethodEntrypointNodeHelper(MethodWithToken targetMethod)
+        public MethodWithGCInfo CreateMethodEntrypointNodeHelper(MethodWithToken targetMethod)
```

Comment:
Is the visibility change needed?

Reply:
No, its an accidental legacy of an earlier variant of the change.